packages/meditrak-app/app/database/schema.js

```diff
@@ -7,7 +7,7 @@ import * as DataTypes from './types';
 
 export const schema = {
   schema: Object.values(DataTypes),
-  schemaVersion: 18,
+  schemaVersion: 19,
```

Comment:
I think schema version 18 for meditrak-app hasn't been released (I've increased from 17 to 18 in one of my recent  meditrak-app task that hasn't been released). So maybe what you just pushed is actually the fix for it?

Reply:
That's a good point, but I'm still ok with this going to 19 (and don't think bumping the app version was the fix)

I think this was the fix for Sepi's test, which was that she had a dev apk based on 18, and was upgrading to the feature branch here which was also 18. It's fine for us to jump 2 schema versions in one release, to make sure testing can happen properly :-)